demo/genome_Snakefile.smk

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env snakemake
+
+'''
+It will only run through the workflow(eg. genome) sub directory.
```

Comment:
Can you please elaborate on this with more complete sentences? This comment does not make this snakefile self-explanatory.

Reply:
what about this now:
"This Snakefile is made to run all the modules compatible with genome workflow.
Compatibility can be checked by referring to the pairing_config parameter present in a default.yaml file of a module."

Reply:
Done